src/Compilers/VisualBasic/Portable/Syntax/InternalSyntax/SyntaxToken.vb

```diff
@@ -453,7 +453,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
 
             If CInt(kind) >= SyntaxKind.AddHandlerKeyword Then
 
-                If CInt(kind) <= SyntaxKind.YieldKeyword OrElse kind = SyntaxKind.NameOfKeyword Then
+                If CInt(kind) <= SyntaxKind.YieldKeyword OrElse
+                        kind = SyntaxKind.NameOfKeyword OrElse
+                        CInt(kind) = SyntaxKind.LoadKeyword _
```

Comment:
4 space indent for consistency.  Also, I don't like it, but the current code pretty consistently keeps the ```Then``` on the same line as the last condition.